content/javascript/concepts/objects/terms/hasOwn/hasOwn.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: '.hasOwn()'
+Description: 'Returns a boolean value based on whether the specified property is directly owned by the object or not.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+Tags:
+  - 'Objects'
+  - 'Methods'
+  - 'Values'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.hasOwn()`** method verifies if a specific property exists in an object, returning true if present, otherwise false. Unlike the `in` operator, it solely inspects direct object properties and doesn't consider inherited ones. This method serves as a replacement for `Object.hasOwnProperty()`.
```

Comment:
```suggestion
The **`.hasOwn()`** method verifies if a specific property exists in an object, returning `true` if present, otherwise `false`. Unlike the `in` operator, it solely inspects direct object properties and doesn't consider inherited ones. This method serves as a replacement for `Object.hasOwnProperty()`.
```